utils/examples/gen_input.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+
+use openvm_example_utils::{
+    hex_str_encode_bytes, hex_str_encode_field_elements, Error, OutputDest,
+};
+use openvm_stark_sdk::{
+    openvm_stark_backend::p3_field::{Field, PrimeField32},
+    p3_baby_bear::BabyBear,
+};
+
+fn generate_input_from_u64() -> Result<(), Error> {
```

Comment:
just checking: will this one be directly usable by fibonacci program? that would be ideal

Reply:
Yes it's directly usable. I'll add a comment.